training/training/routes/image/schemas.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any, Literal, Optional
+from ninja import Schema
+
+
+class LayerParams(Schema):
+    value: str
+    parameters: list[Any]
+
+
+class ImageParams(Schema):
```

Comment:
the schema should support train + test transforms if the user specifies on the FE

Reply:
but these are optional params

Reply:
Can we do this in a seperate PR?

Reply:
ok. that makes sense to me. please create a github issue to track the work @codingwithsurya